cinema/models.py

```diff
@@ -92,9 +96,9 @@ def clean(self):
                 raise ValidationError(
                     {
                         ticket_attr_name: f"{ticket_attr_name} number "
-                        f"must be in available range: "
-                        f"(1, {cinema_hall_attr_name}): "
-                        f"(1, {count_attrs})"
+                                          f"must be in available range: "
+                                          f"(1, {cinema_hall_attr_name}): "
+                                          f"(1, {count_attrs})"
```

Comment:
In the `clean` method of the `Ticket` model, the attributes `count_rows` and `count_seats_in_row` do not exist in the `CinemaHall` model. You should replace `count_rows` with `rows` and `count_seats_in_row` with `seats_in_row` to match the actual attributes of the `CinemaHall` model.